jitsi-media-transform/src/main/kotlin/org/jitsi/nlj/stats/EndpointConnectionStats.kt

```diff
@@ -144,20 +144,30 @@ class EndpointConnectionStats(
             // The delaySinceLastSr value is given in 1/65536ths of a second, so divide it by .000065536 to get it
             // in nanoseconds
             val remoteProcessingDelay = Duration.ofNanos((reportBlock.delaySinceLastSr / .000065536).toLong())
-            rtt = (Duration.between(srSentTime, receivedTime) - remoteProcessingDelay).toDoubleMillis()
-            if (rtt > 7.secs.toMillis()) {
+            if (remoteProcessingDelay > Duration.ofMinutes(5)) {
```

Comment:
How did you pick 5 minutes for this value?  It seems high.

Reply:
I didn't think much about it. I saw 2h35m in logs and it was causing problems. What's a more reasonable value?

Reply:
Did you actually see some large values here (as opposed to the calculated RTT)?